docs/src/pages/[platform]/getting-started/migration/migration.react.mdx

```diff
@@ -213,3 +209,7 @@ please see the [Form Field Customization](../connected-components/authenticator/
 #### CSS Styling
 
 The latest version of the `Authenticator` has a completely different set of CSS variables. Please look over the [Amplify CSS Variables](../connected-components/authenticator/customization#styling) section for more information.
+
+#### `onAuthUiStateChange`
+
+For previous use of `onAuthUiStateChange`, see [useAuthenticator](/react/connected-components/authenticator/headless#useauthenticator-hook).
```

Comment:
I'm concerned that this might just confuse users once they land on the `useAuthenticator` documentation since there is nothing related to `onAuthUiStateChange` in that documentation. 

Suggest thinking about how to help the user understand what they need to use in order to migrate from `onAuthUiStateChange`

Reply:
How about the following? @calebpollman 

```
`onAuthUiStateChange` is deprecated and migrated to `useAuthenticator` since v2, see [useAuthenticator](/react/connected-components/authenticator/headless#useauthenticator-hook).
```

Reply:
left a suggestion below

Reply:
Done d732e33c36d7f3acc899441f606c2b7ea3e533cd